.husky/pre-commit

```diff
@@ -5,19 +5,19 @@
 cd "$(dirname -- "$0")/.." || exit 1
 
 # Ensure node is available (same as asciinema-rec.sh)
-if [[ ! -x "$(command -v node)" ]]; then
-  if [[ ! -x "$(command -v nvm)" && -f ~/.nvm/nvm.sh ]]; then
+if [ ! -x "$(command -v node)" ]; then
```

Comment:
aside you could run volta these days if you werent aware:

https://volta.sh/

makes managing a lot of this _so much easier_

Reply:
I will have to check this out! nvm has typically been great for me, but there are definitely a lot of hacks in here for it.